src/ekka.appup.src

```diff
@@ -0,0 +1,20 @@
+%% -*-: erlang -*-
+
+{"4.2.2",
```

Comment:
we have been using 0.7.4 since emqx v4.2.0
so it should be 0.7.5 here (the tag to be created after merge)
and the upgrade from and downgrade to tags below should be 0.7.4